components/hab/src/cli.rs

```diff
@@ -1458,6 +1459,17 @@ fn valid_ident(val: String) -> result::Result<(), String> {
     }
 }
 
+#[allow(clippy::needless_pass_by_value)] // Signature required by CLAP
+fn valid_ident_or_toml_file(val: String) -> result::Result<(), String> {
+    if is_toml_file(&val) {
+        // We could do some more validation (parse the whole toml file and check it) but that seems
+        // excessive.
```

Comment:
I like this comment in `is_toml_file`, but I dont think it is necessary to duplicate it here.